Discovery/Discovery/Presentation/SearchView.swift

```diff
@@ -68,7 +70,7 @@ public struct SearchView: View {
                     }
                 }
                 .frame(minHeight: 48)
-                .frame(maxWidth: 532)
+                .frame(maxWidth: .infinity)
```

Comment:
How will it look on tablets?

Reply:
It's taking the available space just like list view.

<img width="543" alt="image" src="https://github.com/openedx/openedx-app-ios/assets/5606473/d45067e7-2b0f-4045-8c95-dfa8254646fd">